docs/_articles/en/lwc/linting.md

```diff
@@ -11,37 +11,47 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ## Installation
 
-1. If you created an SFDX project using the `force:project:create` command, the project may contain a `package.json` file with the ESLint plugin already included. Open `package.json` and verify that it contains the following:
+### For a New Project
 
-   ```json
-   "scripts": {
-       "lint": "npm run lint:lwc",
-       "lint:lwc": "eslint force-app/main/default/lwc"
-   }
-   ```
+If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
```

Comment:
included _and configured?_

Reply:
That makes sense, though step 2 is to also include the file with the linting rules.

Reply:
Guess its not configured then so we can leave that off.